lib/stft.dart

```diff
@@ -120,6 +120,23 @@ extension Window on Float64List {
       return 0.42 - 0.5 * math.cos(t) + 0.08 * math.cos(2 * t);
     });
   }
+
+  static Float64List gaussian(
+    int size, {
+    double alpha = 0.25,
+  }) {
+    final res = Float64List(size);
+    final samplingPeriods = (size - 1) * 0.5;
+
+    // standard deviation is alpha * size/2
+    final standardDeviation = (alpha * (size - 1)) * 0.5;
+    for (int i = 0; i < size; i++) {
```

Comment:
Try to use the `_makeWindow` helper, like the other window functions. It's faster because it only evaluates half the window function, then mirrors it over the rest.

Reply:
> Try to use the `_makeWindow` helper, like the other window functions. It's faster because it only evaluates half the window function, then mirrors it over the rest.

Makes sense, thanks!